api/specs/webserver/v0/components/schemas/activity.yaml

```diff
@@ -0,0 +1,27 @@
+ActivityEnveloped:
+  type: object
+  required:
+    - data
+  properties:
+    data:
+      $ref: '#/Activity'
+      additionalProperties: true
```

Comment:
I have some questions
 - do you really need to create separate schemes for ``Activity`` and ``Status``?
 - why to add [Free Form Objects](https://swagger.io/docs/specification/data-models/data-types/) here and not in ``Activity``? Is this even valid? Should you not use an ``AllOf`` and list lines 7 and 8 below?